spring-integration-redis/src/main/java/org/springframework/integration/redis/util/RedisLockRegistry.java

```diff
@@ -524,18 +526,111 @@ private RedisLockRegistry getOuterType() {
 			return RedisLockRegistry.this;
 		}
 
+	}
+
+	private final class RedisPubSubLock extends RedisLock {
+
+		private static final String UNLINK_UNLOCK_SCRIPT =
+				"if (redis.call('unlink', KEYS[1]) == 1) then " +
+				"redis.call('publish', ARGV[1], KEYS[1]) " +
+				"return true " +
+				"end " +
+				"return false";
+
+		private static final String DELETE_UNLOCK_SCRIPT =
+				"if (redis.call('del', KEYS[1]) == 1) then " +
+				"redis.call('publish', ARGV[1], KEYS[1]) " +
+				"return true " +
+				"end " +
+				"return false";
+
+		private static final RedisScript<Boolean> unLinkUnLockScript = new DefaultRedisScript<>(UNLINK_UNLOCK_SCRIPT, Boolean.class);
```

Comment:
If it is `static final`, then it is a constant and therefore has to be declared only in capital letters.